### PR TITLE
(i18n): Updated French translations

### PR DESCRIPTION
**Pull request by loanlaux**
This PR updates French translations for most `core` and `included` plugins, many of which were wrong or clumsy. Any feedback from another French native speaker would be greatly appreciated!

### CODE REVIEW

File: imports/plugins/core/checkout/server/i18n/fr.json [hunk 1]
```diff
@@ -10,13 +10,13 @@
...
         "orderTotal": "Total de la commande",
-        "fulfilledBy": "Réalisé par",
+        "fulfilledBy": "Effectué par",
         "quantityTotal": "Quantité totale",
         "orderSubtotal": "Total partiel",
-        "orderShipping": "livraison",
+        "orderShipping": "Livraison",
         "orderTax": "Paramètres de taxe",
-        "trackYourDelivery": "Les mises à jour concernant la commande seront envoyées à",
-        "registerGuest": "salut! Ajouter un email et recevoir des mises à jour de commande"
```
Comment (prinzdezibel) [inline imports/plugins/core/checkout/server/i18n/fr.json]: @loanlaux shouldn't that start with capital `S`?

File: imports/plugins/core/payments/server/i18n/fr.json [hunk 2]
```diff
@@ -12,21 +12,21 @@
         "dashboard": {
           "paymentsLabel": "Paiements",
           "paymentsTitle": "Paiements",
-          "paymentsDescription": "configuration des paiements"
```
Comment (prinzdezibel) [inline imports/plugins/core/payments/server/i18n/fr.json]: Capital?

File: imports/plugins/core/checkout/server/i18n/fr.json (hunk 1 above)
Comment (prinzdezibel) [reply imports/plugins/core/checkout/server/i18n/fr.json]: sorry. looking at the old stuff :)

File: imports/plugins/core/checkout/server/i18n/fr.json [hunk 3]
```diff
@@ -10,13 +10,13 @@
...
+        "trackYourDelivery": "Les informations concernant la commande seront envoyées à",
+        "registerGuest": "Bonjour ! Renseignez votre adresse e-mail pour être informé de l'avancement de votre commande"
```
Comment (mattt416) [inline imports/plugins/core/checkout/server/i18n/fr.json]: Honest question here -- is it normal in French to put spacing before punctuation like `!`, `?`, etc.?  I see you've done this a fair bit but similar exists in places in the original translations.

Comment (loanlaux) [reply imports/plugins/core/checkout/server/i18n/fr.json]: That's one of the weirdest little-know quirks of the French language! We do put a space before what I'd call "double punctuation" (that is, any punctuation composed of two parts, such as "!", "?", ":" and ";"). However, we don't put spaces before periods or commas.

I'd have a hard time explaining why. I've seen Canadians often do the same though.

Comment (mattt416) [reply imports/plugins/core/checkout/server/i18n/fr.json]: I'll be honest, it does read slightly better and I do it subconsciously from time to time (Canadian here).  Thanks for the info!  :)

**Pull request merged**